integration-tests/__tests__/__snapshots__/coverage_report.test.js.snap

```diff
@@ -48,14 +48,14 @@ File                                 |  % Stmts | % Branch |  % Funcs |  % Lines
 -------------------------------------|----------|----------|----------|----------|----------------|
 All files                            |    56.52 |        0 |       50 |    56.52 |                |
  coverage-report                     |    41.18 |        0 |       25 |    41.18 |                |
-  not-required-in-test-suite.js      |        0 |        0 |        0 |        0 |  8,15,16,17,19 |
-  other-file.js                      |      100 |      100 |      100 |      100 |                |
-  sum.js                             |    85.71 |      100 |       50 |    85.71 |             12 |
-  sum_dependency.js                  |        0 |        0 |        0 |        0 |     8,10,11,14 |
+  OtherFile.js                       |      100 |      100 |      100 |      100 |                |
```

Comment:
updated this snapshot because of file names changing, but if you look, you can see no number changed 👍